sdks/universal-router-sdk/src/utils/commandParser.ts

```diff
@@ -127,3 +148,17 @@ export function parseV3PathExactOut(path: string): readonly V3PathItem[] {
 
   return res
 }
+
+function v4RouterCallToParams(actions: readonly V4RouterAction[]): readonly Param[] {
```

Comment:
open to feedback on this remapping of names. The alternative was to keep `command.params` as a V4RouterAction object like 

```
{
   actionName: SWAP_EXACT_IN,
   ActionType: V4Action.SWAP_EXACT_IN,
   params: [...[
}
```

opted for this to be more standardized with the other non-v4 command datas